operator/src/main/java/org/keycloak/operator/v2alpha1/KeycloakDeployment.java

```diff
@@ -101,60 +99,6 @@ private Deployment fetchExistingDeployment() {
                 .get();
     }
 
-    private void addInitContainer(Deployment baseDeployment, List<String> extensions) {
-        var skipExtensions = Optional
-                .ofNullable(extensions)
-                .map(e -> e.isEmpty())
-                .orElse(true);
-
-        if (skipExtensions) {
-            return;
-        }
-
-        // Add emptyDir Volume
-        var volumes = baseDeployment.getSpec().getTemplate().getSpec().getVolumes();
-
-        var extensionVolume = new VolumeBuilder()
-                .withName(Constants.EXTENSIONS_VOLUME_NAME)
-                .withNewEmptyDir()
-                .endEmptyDir()
-                .build();
-
-        volumes.add(extensionVolume);
-        baseDeployment.getSpec().getTemplate().getSpec().setVolumes(volumes);
-
-        // Add the main deployment Volume Mount
-        var container = baseDeployment.getSpec().getTemplate().getSpec().getContainers().get(0);
-        var containerVolumeMounts = container.getVolumeMounts();
-
-        var extensionVM = new VolumeMountBuilder()
-                .withName(Constants.EXTENSIONS_VOLUME_NAME)
-                .withMountPath(Constants.KEYCLOAK_PROVIDERS_FOLDER)
-                .withReadOnly(true)
-                .build();
-        containerVolumeMounts.add(extensionVM);
-
-        container.setVolumeMounts(containerVolumeMounts);
-
-        // Add the Extensions downloader init container
-        var extensionsValue = extensions.stream().collect(Collectors.joining(","));
-        var initContainer = new ContainerBuilder()
-                .withName(Constants.INIT_CONTAINER_NAME)
-                .withImage(config.keycloak().initContainerImage())
-                .withImagePullPolicy(config.keycloak().initContainerImagePullPolicy())
```

Comment:
Please remove also the unused config options:
https://github.com/keycloak/keycloak/blob/1f10a4feb2dfea88bb62e76e314c4e999ef75cc9/operator/src/main/resources/application.properties#L11-L12

Reply:
👍  done